test/test_users.py

```diff
@@ -37,4 +38,32 @@ def test_log_in(client,test_user):
 ])
 def test_incorrect_log_in(client, email, password, status_code):
     res = client.post("/api/v1/users/login", data={"username": email, "password": password})
-    assert res.status_code == status_code
\ No newline at end of file
+    assert res.status_code == status_code
+    
+    
+def test_get_user_list(client):
+    
+    post_request = client.post(
+      "/api/v1/users/register/", 
+      json={
+          "first_name": "Slightly",
+          "last_name": "Techie", 
+          "email": "slightlytechie@gmail.com", 
+          "password": "food", 
+          "password_confirmation": "food"
+        }
+    )
+    new_user = user.UserResponse(**post_request.json())
+    
+    get_request = client.get("/api/v1/users")
+    
+    # topology of the final response unknown yet.
+    # uncertain if status_code will be included with the list of users
```

Comment:
Did you run this test locally to see if there are any errors?

Looks like you're trying to fetch status code from a dictionary